### ISSUE TITLE
Server crash - Placing a hopper under a seed analyzer pointing into a seed chest.

### ISSUE THREAD

**Issue by ProsperCraft** [InfinityRaider/AgriCraft#448]
java.lang.NullPointerException: Ticking block entity
at net.minecraft.tileentity.TileEntityHopper.func_152102_a(TileEntityHopper.java:429)

http://pastebin.com/n5G3xJHV

**Comment by InfinityRaider** [comment]
Cant seem to reproduce this one, what kind of seed was in it?

**Comment by ProsperCraft** [comment]
Are you trying on a client/server setup?

Also, fastcraft and kcauldron are running.

he suspects it was a rutabegga seed.

**Comment by InfinityRaider** [comment]
Yes: http://puu.sh/l9H6u/65da32a15f.png

**Comment by ProsperCraft** [comment]
Seed analyzer on top of the hopper..

**Comment by InfinityRaider** [comment]
Derp, my bad.

**Comment by InfinityRaider** [comment]
Still: http://puu.sh/laoui/84b06f319a.png

Does this happen every time, or was it a singular occurance? The crash log is obfuscated code and is quite hard to debug if I can't replicate it inside my dev env.

**Comment by VT-14** [comment]
Both pictures of InfinityRadar's setup point into normal chests.  I interpreted the title as the hopper pointing into a seed storage (block).
Agricraft 1.4.3
Minecraft 1.7.10
Forge 10.13.4.1541

![2015-11-09_15 51 51](https://cloud.githubusercontent.com/assets/12564251/11046199/fec3f422-86f9-11e5-8fee-8d2a4dab5c6e.png)

That crashes if there is a journal in the seed analyzer after analyzing seeds.  It didn't crash when there was no journal.  Tested with potato and the iron seeds.

Also, when used with no journal: it didn't crash, but the seed was lost.

Crashlog: https://gist.github.com/VT-14/2f7dca44ac5e0ef313c7
Normal Log: https://gist.github.com/VT-14/df998349ab74b08ce931
